x-pack/plugins/security_solution/public/app/home/template_wrapper/bottom_bar/index.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable react/display-name */
+
+import React, { useRef } from 'react';
+import { KibanaPageTemplateProps } from '../../../../../../../../src/plugins/kibana_react/public';
+import { AppLeaveHandler } from '../../../../../../../../src/core/public';
+import { useKibana } from '../../../../common/lib/kibana';
+import { useShowTimeline } from '../../../../common/utils/timeline/use_show_timeline';
+import { useSourcererScope } from '../../../../common/containers/sourcerer';
+import { DETECTIONS_SUB_PLUGIN_ID } from '../../../../../common/constants';
+import { SourcererScopeName } from '../../../../common/store/sourcerer/model';
+import { TimelineId } from '../../../../../common/types/timeline';
+import { AutoSaveWarningMsg } from '../../../../timelines/components/timeline/auto_save_warning';
+import { Flyout } from '../../../../timelines/components/flyout';
+
+export const BOTTOM_BAR_CLASSNAME = 'timeline-bottom-bar';
+
+export const SecuritySolutionBottomBar = React.memo(
```

Comment:
@semd be aware of that

Reply:
@XavierM Sure. We'll have to use the path as Angela did in her part

Reply:
you meant we still have to check the path here to decide the sourcerer scope right?